Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```diff
@@ -167,6 +167,8 @@
   - type: Item
     sprite: Objects/Weapons/Guns/Shotguns/enforcer_inhands_64x.rsi
   - type: BallisticAmmoProvider
+    soundRack: # Starlight
+      path: /Audio/_Starlight/Weapons/Guns/Cock/Shotgun-Rack.ogg
```

Comment:
this sound effect, while mono, has high peaks (manages to max out my headphones) which makes it kind of painful to listen to at the same volume as the rest of the sound effects ingame - this can be fixed by either reducing the "volume" through yaml, or running your sound effect through audacity to compress the peaks to something a bit quieter while maintaining the rest of the sound